src/ipfs/validateIPFSCid.ts

```diff
@@ -0,0 +1,11 @@
+import { CID } from 'multiformats';
+
+/**
+ * Wrapper method for IPFS CID string validation
+ *
+ * @param ipfsCid - The CID string to validate.
+ * @throws Throw an error if CID is not valid
+ */
+export const validateIPFSCid = (ipfsCid: string): void => {
+  CID.parse(ipfsCid);
+};
```

Comment:
I tried export `validateIPFSCid` (see [sdk@0.7.0-rc2](https://www.npmjs.com/package/@nexusmutual/sdk/v/0.7.0-rc2)) but its causing issues because `multiformats` is strictly ESM and it breaks when its imported in a JS project that uses CommonJS (`require` to import it.)

So in the interest of making our sdk fully compatible for both CommonJS (require) and ESM (import), we're not including it on SDK exported functions

Reply:
i've downgraded multiformats to v8 which is the last version that is compatible with CommonJS. i can confirm the issue is fixed